src/ui/Analytics/Analytics.ts

```diff
@@ -136,7 +137,17 @@ export class Analytics extends Component {
      * Default value is `undefined`, and the value of this parameter will fallback to the organization used for the
      * search endpoint.
      */
-    organization: ComponentOptions.buildStringOption()
+    organization: ComponentOptions.buildStringOption(),
+
+    /**
+     * Sets the Origin Context dimension on the analytic events.
+     *
+     * You can use this dimension to specify the context of your application.
+     * Suggested values are "Search", "InternalSearch" and "CommunitySearch"
+     *
+     * Default value is `Search`.
+     */
+    originContext: ComponentOptions.buildStringOption({ defaultValue: 'Search' }),
```

Comment:
It is my understanding that we want this to be `hidden` so to speak, so that it does not change from deployment to deployment. Is this still true ?

As in we want a "secret" value that is "harcoded" by the integration.

Then, do we really need to expose this as an option on the component ?

Does it need to be documented, and available in the public reference documentation ?